packages/css/src/types.ts

```diff
@@ -1,8 +1,3 @@
-/**
- * Copied and adapted from @types/styled-system__css
- * https://github.com/DefinitelyTyped/DefinitelyTyped/blob/028c46f833ffbbb0328a28ae6177923998fcf0cc/types/styled-system__css/index.d.ts
```

Comment:
The changes here are pretty noticeable already, I'd say. We probably shouldn't link to "the source" anymore.

Reply:
Agreed.